src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/Commons.cs

```diff
@@ -163,6 +164,10 @@ internal static class NNStreamer
     {
         internal const string TAG = "ML.Inference";
 
+        internal const string FeatureKey = "http://tizen.org/feature/machine_learning.inference";
+
+        internal const string CAPINnstreamerPath = "/usr/lib/libcapi-nnstreamer.so.0";
```

Comment:
The library path can be different in x64 systems. Is it enough checking feature? 
If there is no case of the feature is "on" and the so file is not exist, the file existing checking is not necessary.

Reply:
ACK! I will use an alternative solution instead of fixed path.